compiler/src/dotty/tools/dotc/core/CheckRealizable.scala

```diff
@@ -50,7 +50,15 @@ object CheckRealizable {
   private val LateInitialized = Lazy | Erased
 }
 
-/** Compute realizability status */
+/** Compute realizability status.
```

Comment:
@abeln You asked questions about stability and realizability, do you have a chance to take a look at this comment?